packager/app/packager_main.cc

```diff
@@ -107,7 +107,10 @@ const char kUsage[] =
     "    The group ID for the output stream. For HLS this is used as the\n"
     "    GROUP-ID attribute for EXT-X-MEDIA.\n"
     "  - playlist_name: Required for HLS output.\n"
-    "    Name of the playlist for the stream. Usually ends with '.m3u8'.\n";
+    "    Name of the playlist for the stream. Usually ends with '.m3u8'.\n"
+    "  - skip_encryption: Optional. If this is set to a non-zero value,\n"
```

Comment:
This description does not match what is implemented, rephrase it as:

- skip_encryption=0|1: Optional. Default to 0 if not specified. If it is set to 1, ...